rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -659,6 +665,17 @@ void main_window::InstallPackages(QStringList file_paths)
 		const QString pattern = QString(".*\\.%1").arg(QString::fromStdString(extension));
 		for (const auto& file : file_paths.filter(QRegExp(pattern, Qt::CaseInsensitive)))
 		{
+			if (!m_gui_settings->GetBootConfirmation(this))
+			{
+				// Last chance to cancel the operation
+				return;
```

Comment:
seems pointless in a lambda.
I suggest returning false and then early out if either calls are false

Reply:
wait... just move that section before the lambda